tolerance-review/AppFlow.adoc

```diff
@@ -0,0 +1,38 @@
+:gls_prefix:
+
+// Do not modify section headers, as they are used by flamel and the translation process.
+= pass:a,n[{gls_res_outcomes}]
+
+You should be able to apply different fault tolerance policies to Quarkus applications given identified potential faults.
+
+== Present App Flow
+
+The present lab makes use of Session, Speaker and Schedule services.
+
+* `Aliveness` and `Readiness` checks are implemented.
```

Comment:
I think no one has picked the health checks section yet.
Should we wait until someone is working in this section and can provide feedback?

Reply:
I guess we can provide our feedback and it is open to changes later too.